packages/jest/src/plugins/plugin.ts

```diff
@@ -124,13 +119,7 @@ async function buildJestTargets(
   const absConfigFilePath = resolve(context.workspaceRoot, configFilePath);
 
   if (require.cache[absConfigFilePath]) {
-    for (const k of Object.keys(require.cache)) {
-      // Only delete the cache outside of jest-validate
-      // jest-validate has a Symbol which is important for jest config validation which breaks if the require cache is broken
-      if (relative(jestValidatePath, k).startsWith('../')) {
-        delete require.cache[k];
-      }
-    }
```

Comment:
It might not be a bad idea to keep this one. I think for the `jest` plugin, it can be selective. The others, we can change after plugin isolation is always enabled.

Reply:
I don't think it hurts too much to leave the require cache in place for node_modules, being selective only really means that things will be a bit slower after all. Who's to say some jest plugin doesn't have a symbol 🤷 

We can chat more about it if you'd like, but I think it makes sense to just use the same method other plugins do.